index.html

```diff
@@ -38,5 +38,181 @@ <h1 class="lines"> Hey there <br>
           <img src="images/vector.svg" alt="normalbottun">
         </div>
     </section>
+    
+
+    <section class="line-break">
+      <div>
+      <h2>My Recent <br> Works</h2>
+      </div>
+    </section>
+
+
+    <section class="section-placeholder">
+      <div  class="placeholder-container">
+          <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+          <div class="text-container">
+            <h1>Multi-Post Stories <br> Gain+Glory</h1>
```

Comment:
- [ ] Kindly use another heading like h2 ...... hx for subtitles. Use h1 only for the main Title.